cmd/transportRequestUploadSOLMAN.go

```diff
@@ -5,16 +5,14 @@ import (
 	"github.com/SAP/jenkins-library/pkg/log"
 	"github.com/SAP/jenkins-library/pkg/piperutils"
 	"github.com/SAP/jenkins-library/pkg/telemetry"
+	"github.com/SAP/jenkins-library/pkg/transportrequest"
 	"github.com/SAP/jenkins-library/pkg/transportrequest/solman"
 )
 
 type transportRequestUploadSOLMANUtils interface {
-	command.ExecRunner
-
+	solman.Exec
```

Comment:
What does this do?

Reply:
from common.go. it is the interface:
```
type Exec interface {
	command.ExecRunner
	GetExitCode() int
}
```

used as parameter of 

`	Perform(command Exec) (string, error)
`